service/src/test/java/bio/terra/cbas/dao/TestMethodDao.java

```diff
@@ -50,22 +58,40 @@ class TestMethodDao {
           methodSource,
           workspaceId);
 
+  @Container
+  static JdbcDatabaseContainer postgres =
+      new PostgreSQLContainer("postgres:14")
+          .withDatabaseName("test_db")
+          .withUsername("test_user")
+          .withPassword("test_password");
+
+  @DynamicPropertySource
+  static void postgresProperties(DynamicPropertyRegistry registry) {
+    registry.add("spring.datasource.jdbc-url", postgres::getJdbcUrl);
+    registry.add("spring.datasource.username", postgres::getUsername);
+    registry.add("spring.datasource.password", postgres::getPassword);
+  }
```

Comment:
This potentially looks like a good candidate for extracting up into an abstract class that all of these DAO tests can inherit? Is that possible?